### DISCUSSION TITLE
WPILib 2024

### DISCUSSION THREAD

**Discussion by uwuscutely** [BroncBotz3481/YAGSL-Example#120] [General]
Hey I was wondering if a port of the example project to WPILib 2024 was in the works somewhere

**Comment by thenetworkgrinch** [comment]
It is in the works, I am waiting on CTRE 2024 and ReduxLib 2024 official releases. The beta project is here

https://github.com/BroncBotz3481/YAGSL-Example/tree/2024/src%2Fmain%2Fjava%2Fswervelib

I am also working on more docs at https://yagsl.gitbook.io/yagsl

**Reply by uwuscutely** [reply]
Awesome, thank you for the update and the hard work!

**Comment by tfield** [comment]
Phoenix 6 libs are out and cancoder firmware 24 is out - so far so good!

**Comment by thenetworkgrinch** [comment]
It is getting released right now

**Comment by IgorCarneiroDaSilva** [comment]
when do you think the 2024 example is coming out?

**Reply by thenetworkgrinch** [reply]
Its already out... and updated in this repository
